docs/writing-tests.md

```diff
@@ -1,32 +1,12 @@
 # Writing tests
 
-## Adding new tests
+## Alerting in Icinga
 
-Tests that are supposed to be run by Icinga also have to be added to the file
-`modules/monitoring/manifests/checks/smokey.pp` in the [govuk-puppet](https://github.com/alphagov/govuk-puppet) repository. For
-example, the test [frontend.feature](/features/frontend.feature)
-is added to Icinga like this:
+In order to alert on a failing scenario in Icinga, you need to:
 
-```puppet
-icinga::check_feature {
-  'check_frontend':          feature => 'frontend';
-  #other feature tests
-}
-```
+- Ensure the feature is [listed in the Puppet config](https://github.com/alphagov/govuk-puppet/blob/d7af16e96aed682facb5cf5bc3e3972510c64ca2/hieradata_aws/integration.yaml#L378)
 
-## Prioritising scenarios
+- Tag the scenario using "@<tag>" (tags are listed [here](https://github.com/alphagov/govuk-puppet/blob/4ec2949332079570be5493753bb84a983218a444/modules/icinga/manifests/check_feature.pp))
 
-Because we integrate Icinga with the output from these tests, we provide a set
-of tags which match with how important we consider a scenario to be. `@high` and
-above will trigger pager alerts.
-
-Each scenario can and should be prioritised by using the `@urgent`, `@high`,
-`@normal` or `@low` cucumber tags. For example, the frontend scenario "check
-guides load" can be prioritised like this:
-
-```cucumber
-@low
-Scenario: check guides load
-  When I visit "/getting-an-mot/overview"
-  Then I should see "Getting an MOT"
-```
+  - `@urgent` tags will trigger will [trigger out-of-hours callouts](https://github.com/alphagov/govuk-puppet/blob/51d7f7b648e8fde5aae3adfb774ec3bca6325bd8/modules/monitoring/manifests/contacts.pp#L163)
+  - Other tags are purely to help us prioritise fixing things
```

Comment:
Thanks for providing some guidance on prioritisation. I wonder if we should do more on this at some point (out of scope for this PR).

What makes a smokey test _low_ priority? 🤔  The deleted "Getting an MOT" example seems high priority to me, since the absence of the text suggests the guidance is missing.

Reply:
Yeah, I briefly looked into this and found it's arbitrary. I'm not sure we need more categories than just "@urgent", since otherwise the behaviour is basically the same. It would be nice to get rid of stuff we don't need, especially when it has no clear reasoning behind it - something for the future!

Reply:
Shall we live the dream? (https://github.com/alphagov/smokey/pull/693)